GVFS/GVFS.Common/Database/IPlaceholderData.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace GVFS.Common.Database
+{
+    public interface IPlaceholderData
+    {
+        string Path { get; }
+        string Sha { get; }
+        bool IsFolder { get; }

```

Comment:
Rename to `IsPartialFolder`?

Reply:
Are all three of these `bool`s independent? Should they be an `enum` instead?

Reply:
Looking at this closer and `IsFolder` should stay because it really is if the placeholder is any type of folder and not just a partial folder.
```
public bool IsFolder
{
	get
	{
		return IsShaAFolder(this.Sha);
	}
}


public static bool IsShaAFolder(string shaValue)
{
	return shaValue.Equals(PartialFolderValue, StringComparison.Ordinal) ||
		shaValue.Equals(ExpandedFolderValue, StringComparison.Ordinal) ||
		shaValue.Equals(PossibleTombstoneFolderValue, StringComparison.Ordinal);
}
```